ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigPhase0.java

```diff
@@ -116,6 +116,10 @@ public class SpecConfigPhase0 implements SpecConfig {
   private final int attestationSubnetCount;
   private final int attestationSubnetExtraBits;
   private final int attestationSubnetPrefixBits;
+  private final int reorgMaxEpochsSinceFinalization;
+
```

Comment:
Any reason for this empty line?

Reply:
hrm not really, i can clean that up, i just did the first one then the second two